lib/_http_server.js

```diff
@@ -625,7 +616,7 @@ function socketOnData(server, socket, parser, state, d) {
 function onRequestTimeout(socket) {
   socket[kRequestTimeout] = undefined;
   // socketOnError has additional logic and will call socket.destroy(err).
-  ReflectApply(socketOnError, socket, [new ERR_HTTP_REQUEST_TIMEOUT()]);
+  Reflect.apply(socketOnError, socket, [new ERR_HTTP_REQUEST_TIMEOUT()]);
```

Comment:
Note fore me, `ReflectApply is not actualy needed here, we can avoid an array allocation.